### ISSUE TITLE
Stremio Version: Theater v1.3.0 and OS: Tizen (Samsung Tv) problems with Romanian subtitle! 

### ISSUE THREAD

**Issue by Radoo02** [Stremio/stremio-bugs#996]
Stremio Version
Stremio Version: Theater v1.3.0 and OS: Tizen (Samsung Tv)

Is your feature request related to a problem? Please describe.
On a Samsung TV, Tizen platform, I choose the default Romanian language subtitle. This does not appear automatically and I have to enter it manually every time. I want to specify that if I choose any other language, for example English, German, they remain set and the subtitle appears automatically. This problem only occurs in the Romanian language.

Describe the solution you'd like
I want the language to remain set and the subtitle to enter automatically.

Thanks!

**Comment by TRtomasz** [comment]
Please give example of a stream for which this is happening

**Comment by Radoo02** [comment]
All the streaming have the same problem!

**Comment by TRtomasz** [comment]
yes thats a known issue @Dennis-NL  it will be fixed with the next release we are awaiting samsung store approval

**Comment by Dennis-NL** [comment]
Can i download it somewhere? To install with Tizen studio

**Comment by TRtomasz** [comment]
its not possible, each app has to be signed by samsung certificate or certifcate that is assigned to your TV SN

**Comment by Dennis-NL** [comment]
Yes but i can generate it myself? Works with Jellyfin

**Comment by TRtomasz** [comment]
You cant, app is not opensourced at the moment

**Comment by Dennis-NL** [comment]
How long can approval take?

**Comment by kKaskak** [comment]
> > yes thats a known issue @Dennis-NL it will be fixed with the next release we are awaiting samsung store approval
> 
> How long does this take? Surely approval does not take 2 months right?

Approval takes sometimes up to 4 months. Both for Samsung and LG.
